php/conf.d/custom.ini

```diff
@@ -40,6 +40,8 @@ opcache.memory_consumption = 256
 opcache.save_comments = On
 opcache.revalidate_freq = 0
 opcache.validate_timestamps = On
+opcache.file_cache = /tmp/
```

Comment:
Same as before, I think that these two settings should be defined for CLI only.

By the way, please keep the alphabetical order. 😉